Resources/skeleton/crud/actions/delete.php.twig

```diff
@@ -10,7 +10,8 @@
     public function deleteAction({{ entity }} ${{ entity|lower }}, Request $request)
     {
         $form = $this->createDeleteForm(${{ entity|lower }}->getId(), '{{ route_name_prefix }}_delete');
-        if ($form->handleRequest($request)->isValid()) {
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
```

Comment:
Can we just keep it in one line? `if ($form->handleRequest($request)->isSubmitted() && $form->isValid())`

Reply:
I did it the way Symfony suggests in its documentation.
I suppose it could be done in one line, but I'm not sure why that would be preferrable.

Reply:
Because one line is less than two lines

Reply:
ok